view/range/index.css

```diff
@@ -12,11 +12,17 @@ body {
   &.is-a {
     background: var(--opacity-pattern);
     border: none;
+
+    &::before {
```

Comment:
According to my code style, `::before` rule should be at separated line (not nested).

I am thinking of `::before` as an element.